apps/query-eval/queryeval/driver.py

```diff
@@ -239,7 +241,7 @@ def do_plan(self, query: QueryEvalQuery, result: QueryEvalResult) -> QueryEvalRe
                 self.config.config.index,
                 self.data_schema,
                 examples=self.examples or None,
-                natural_language_response=self.config.config.natural_language_response or False,
+                natural_language_response=self.config.config.natural_language_response or True,
```

Comment:
This might've broken an integ test, can you validate? Can merge after fixing or confirming